lib/generate/excludedModules.js

```diff
@@ -17,4 +17,18 @@ module.exports = [
     'exports',
     'require',
     'module',
+    'jquery/fileUploader/jquery.fileupload',
+    'jquery/fileUploader/jquery.fileupload-process',
+    'jquery/fileUploader/jquery.fileupload-video',
+    'jquery/fileUploader/jquery.fileupload-audio',
+    'jquery/fileUploader/jquery.fileupload-image',
+    'jquery/file-uploader',
+    'jquery/fileUploader/jquery.iframe-transport',
+    'jquery/fileUploader/vendor/blueimp-load-image/js/load-image',
+    'jquery/fileUploader/vendor/blueimp-load-image/js/load-image-meta',
+    'jquery/fileUploader/vendor/blueimp-load-image/js/load-image-scale',
+    'jquery/fileUploader/vendor/blueimp-load-image/js/load-image-orientation',
+    'jquery/fileUploader/vendor/blueimp-load-image/js/load-image-exif',
+    'jquery/fileUploader/vendor/blueimp-canvas-to-blob/js/canvas-to-blob',
+    'jquery/fileUploader/vendor/jquery.ui.widget',
```

Comment:
We can just blanket exclude everything relating to the two, IMO. 

Please add a comment explaining what this is doing as well – be technical and specific (i.e. discuss the semicolon)

```suggestion
    'jquery/fileUploader/',
    'jquery/file-uploader',
```